packages/react-storage/src/components/StorageBrowser/views/Controls/Table.tsx

```diff
@@ -154,24 +155,28 @@ export function TableControl<U>({
   });
 
   return (
-    <Table
-      {...dropHandlers}
-      aria-label={ariaLabel}
-      data-testid="storage-browser-table"
-      className={`${BLOCK_NAME} ${
-        dragState !== 'inactive' ? `${BLOCK_NAME}__dropzone` : ''
-      }`}
-    >
-      <TableHead className={`${BLOCK_NAME}__head`}>
-        <TableRow className={`${BLOCK_NAME}__row`}>
-          {columns.map((column) => renderHeaderItem(column))}
-        </TableRow>
-      </TableHead>
-
-      <TableBody className={`${BLOCK_NAME}__body`}>
-        {data?.map((row: U, rowIndex: number) => renderRowItem(row, rowIndex))}
-      </TableBody>
-    </Table>
+    <ViewElement className={'storage-browser__table-wrapper'}>
```

Comment:
Double quotes? Also why is this needed?

Reply:
🍕

Reply:
⭐ suck

Reply:
Is Moriahhh Liking pizza every night

Reply:
This is needed to get the table scrolling to work properly